corrected parser error/configuration for "vm endpoint acl-rule create"

ghuntley opened the pull request:
Hey Guys,

This pull request resolves #1500 whereby `--description` on `vm endpoint acl-rule create` would crash the command line tools. The simple omission of `<description>` resulted in your parser passing in a `bool` instead of a `string` into the description validation logic.

Regards,
Geoff

(contribution as per CLA; documents are signed)

abeerag commented:
Could this be reviewed/merged? It causes an annoying bug.

amarzavery commented:
It is against the master branch instead of dev. We shall fix this in upcoming release.

ghuntley commented:
Yo @amarzavery  / @abeerag ; I've reworked this unit of work under the /dev/ branch.

Updated pull request is located at https://github.com/Azure/azure-xplat-cli/pull/1607 and seeks approval/itnegration.
